docs/source/guides/ethereum_node.rst

```diff
@@ -229,3 +240,32 @@ to indicate which are the automated rules (in our case, allowing the listing of
 .. code:: bash
 
     $ clef --keystore <PATH TO KEYSTORE> --chainid <CHAIN ID> --advanced --rules rules.js
+
+
+Usage
++++++
+
+Once ``clef`` is running, specify the Clef signer either through the CLI (``--signer``) or
+API (``nucypher.blockchain.eth.signers.Signer.from_signer_uri``), using the URI ``clef://<CLEF IPC PATH>``.
+
+
+Signing with Local Keystore
+***************************
+
+.. important::
+
+    Can only be used for :ref:`Worker (Ursula) <ursula-config-guide>` operations.
+
+Local keystore signing utilizes `eth-account <https://github.com/ethereum/eth-account>`_ to sign ethereum transactions
+using local ethereum keystore files. By default on Linux, the keystore directory path is ``~/.ethereum/keystore``
+(on MacOS for Goerli testnet, ``/Users/<username>/Library/Ethereum/keystore``).
```

Comment:
```suggestion
(on MacOS, ``/Users/<username>/Library/Ethereum/keystore``).
```

Reply:
See https://github.com/nucypher/nucypher/pull/1945#discussion_r419611966 - that being said, the path is incorrect.